src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -2428,7 +2468,7 @@ static sd_dhcp_client *dhcp_client_free(sd_dhcp_client *client) {
 
 DEFINE_TRIVIAL_REF_UNREF_FUNC(sd_dhcp_client, sd_dhcp_client, dhcp_client_free);
 
-int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize) {
+int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize, bool rapid_commit) {
```

Comment:
Please do not introduce another boolean flag to sd_dhcp_client_new(), but please introduce a setter, e.g. sd_dhcp_client_set_rapid_commit(). Please see sd_dhcp6_client_set_rapid_commit() for DHCPv6.